test/unit/lib/api-client.js

```diff
@@ -0,0 +1,325 @@
+            port: 8080,
+            method: 'post',
+            endpoint: '/api/item',
+            response
+        }).then(() => {
+            return createItem('m8y7nEtAe');
+        }).then(data => {
+            assert.deepEqual(data, response.data);
+            MockServer.validate({
+                modelAddress: 'm8y7nEtAe'
+            });
+        });
+    });
+
+    it('createModel', () => {
+        let response = {
+            status: 'success',
+            data: {
+                address: 'm8y7nEtAe',
+                name: 'Resistor',
+                description: 'V = IR',
+                manufacturer: 'Live',
+                vendor: 'Mouzer',
+                location: 'Shelf 14',
+                isFaulty: false,
+                faultDescription: '',
+                price: 10.50,
+                count: 20
+            }
+        };
+        return MockServer.listen({
+            port: 8080,
+            method: 'post',
+            endpoint: '/api/model',
+            response
+        }).then(() => {
+            return createModel('Resistor', 'V = IR', 'Live', 'Mouzer', 'Shelf 14', false, '', 10.50, 20);
+        }).then(data => {
+            assert.deepEqual(data, response.data);
+            MockServer.validate({
+                name: 'Resistor',
+                description: 'V = IR',
+                manufacturer: 'Live',
+                vendor: 'Mouzer',
+                location: 'Shelf 14',
+                isFaulty: false,
+                faultDescription: '',
+                price: 10.50,
+                count: 20
+            });
+        });
+    });
+
+    it('deleteItem', () => {
+        let response = {
+            status: 'success',
+            data: {
+                items: [{
+                    address: 'iGwEZVHHE',
+                    modelAddress: 'm8y7nEtAe',
+                    status: 'AVAILABLE'
+                }],
+                modelName: 'Resistor'
+            }
+        };
+        return MockServer.listen({
+            port: 8080,
+            method: 'delete',
+            endpoint: '/api/item',
+            response
+        }).then(() => {
+            return deleteItem({
+                address: 'iGwEZUvfA',
+                modelAddress: 'm8y7nEtAe'
+            });
+        }).then(data => {
+            assert.deepEqual(data, response.data);
+            MockServer.validate({
+                itemAddress: 'iGwEZUvfA',
+                modelAddress: 'm8y7nEtAe'
+            });
+        });
+    });
+
+    it('getAllItems', () => {
+        let response = {
+            status: 'success',
+            data: {
+                items: [{
+                    address: 'iGwEZVHHE',
+                    modelAddress: 'm8y7nEtAe',
+                    status: 'AVAILABLE'
+                }]
+            }
+        };
+        return MockServer.listen({
+            port: 8080,
+            method: 'get',
+            endpoint: '/api/item/all',
```

Comment:
I noted this in Slack. Might make a hotfix PR with all of the documentation issues I noted.